src/udp_tracker_connection.cpp

```diff
@@ -154,7 +154,7 @@ namespace libtorrent {
 #endif
 
 		// pick another target endpoint and try again
-		m_target = pick_target_endpoint();
+		m_target = udp::endpoint(m_endpoints.front().address(), m_endpoints.front().port());
```

Comment:
i seem to recall there is a utility function already for converting between udp and tcp endpoints.

Reply:
I cannot find such a function.